src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKAnnotationPluginDescriptorUnitTest.java

```diff
@@ -0,0 +1,575 @@
+        Assert.assertEquals(((InbreedingCoeff) badArgumentsOverridingGoodArguments.get(0)).annotate(null, inbreedingCoefficientVC, null), Collections.emptyMap());
+    }
+
+    //TODO This test is intended to show that the disabling and reenabling an annotation will restore global defaults but unfortunately
+    //     this is not the case due to an open issue having to do with instantiating only one annotation object.
+    //     See https://github.com/broadinstitute/gatk/issues/3848 for the same issue in read filters.
+    @Test (enabled = false)
+    public void testDisableDefaultsAndReplaceOnCommandLine() {
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(Arrays.asList(new InbreedingCoeff(Sets.newSet("s1", "s2", "s3", "s4", "s5", "s6", "s7", "s8", "s9", "s10")),
+                        new DepthPerSampleHC()), Collections.singletonList(StandardAnnotation.class))),
+                Collections.emptySet());
+        List<String> args = Stream.of("-G", StandardAnnotation.class.getSimpleName(), "--"+ StandardArgumentDefinitions.DISABLE_TOOL_DEFAULT_ANNOTATIONS).collect(Collectors.toList());
+
+        clp.parseArguments(nullMessageStream, args.toArray(new String[args.size()]));
+        VariantAnnotatorEngine vae = new VariantAnnotatorEngine(instantiateAnnotations(clp), null, Collections.emptyList(), false);
+
+        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().stream().noneMatch(a -> a.getClass().getSimpleName().equals(DepthPerSampleHC.class.getSimpleName())));
+        Assert.assertTrue(vae.getInfoAnnotations().stream().anyMatch(a -> a.getClass().getSimpleName().equals(BaseQualityRankSumTest.class.getSimpleName())));
+
+        // Asserting that the default set InbreedingCoeff has been overridden
+        Assert.assertEquals(Double.valueOf((String) ( vae.annotateContext(inbreedingCoefficientVC, new FeatureContext(),null, null, s -> true))
+                        .getAttribute(GATKVCFConstants.INBREEDING_COEFFICIENT_KEY)),
+                -0, 0.001, "InbreedingCoefficientScores");
+    }
+
+    @Test
+    public void testNoAnnotations(){
+        CommandLineParser clp = new CommandLineArgumentParser(
+                new Object(),
+                Collections.singletonList(new GATKAnnotationPluginDescriptor(null, null)),
+                Collections.emptySet());
+        String[] args = {};
+        clp.parseArguments(nullMessageStream, args);
+        List<Annotation> annots = instantiateAnnotations(clp);
+        final VariantAnnotatorEngine vae = new VariantAnnotatorEngine(annots, null, Collections.emptyList(), false);
+        Assert.assertTrue(vae.getGenotypeAnnotations().isEmpty());
+        Assert.assertTrue(vae.getInfoAnnotations().isEmpty());
+    }
+
+    @Test
+    public void testGetClassForPluginHelp(){
```

Comment:
This is a negative test I think, so that should be reflected in the name somehow, i.e., `testGetClassForPluginHelpFails`, and then add a positive test that verifies that it works when provided a valid name.

Reply:
It tests both in the test